### PR TITLE
Multiple additions to notify trending

### PR DESCRIPTION
**Pull request by ethervoid**
* Reapply the notify trending deploy because of the failed revert made 2 days ago
* Added simulation message to now what mode are we using

### CODE REVIEW

**Comment by ethervoid** [comment]
Please @Kartones check this PR

File: app/mailers/user_mailer.rb [hunk 1]
```diff
@@ -85,4 +85,18 @@ def table_liked(canonical_visualization, viewer_user, visualization_preview_imag
          :subject => @subject
   end
 
+  def trending_map(visualization, mapviews, visualization_preview_image)
+    @user = visualization.user
+    @bcc = "subscriptions@cartodb.com"
```
Comment (Kartones) [inline app/mailers/user_mailer.rb]: This must be a config param read from `Cartodb.config`, remember opensource & custom installs...

Comment (Kartones) [inline app/mailers/user_mailer.rb]: I'd also implement either a sampling rate or a semaphore (with the PlatformLimits should be easy) to not spam and only send 1 of each X maps, but that's pure personal taste, for starting is ok as it is.

**Comment by Kartones** [comment]
Apart from the hardcoded email, everything else is fine, so change that and you have my :+1:

**Comment by ethervoid** [comment]
Removed that part in order to deploy this asap because we need it working

**Pull request merged**